### DISCUSSION TITLE
Embedding a Drawer in another container

### DISCUSSION THREAD

**Discussion by JasonWeise** [mantinedev/mantine#5598] [Q&A]
I am trying to build custom navbar solution that is able to have a drawer component "embedded" in another container.
Regardless of what I try, I cannot seem to get the Drawer to only fit in the parent container but rather still takes entire height of the page.

What I am trying to acheive is something similar to the following in MudBlazor:
https://mudblazor.com/components/drawer#variants-mini

You can see that the drawer is contained within a parent container.

Has anyone managed to make something like this work in Mantine?

Thanks in advance!

**Comment by rtivital** [comment]
Drawer has a fixed position, you can change it to absolute with Styles API

**Reply by JasonWeise** [reply]
Hey there @rtivital , thanks for replying.
I tried that based on another discussion I saw but it doesn't seem to work.

If I use styles prop and set to absolute it (the Drawer) still expands to the entire height of the screen regardless of the height of its parent.
